scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -815,10 +794,9 @@ public InternalJob executeWork(Session session) {
 
     @SuppressWarnings("unchecked")
     private Map<Long, List<TaskData>> loadJobsTasks(Session session, List<Long> jobIds) {
-        Query tasksQuery = session
-                .createQuery(
-                        "from TaskData as task left outer join fetch task.dependentTasks where task.id.jobId in (:ids)")
+        Query tasksQuery = session.getNamedQuery("loadJobsTasks")
```

Comment:
Maybe a javadoc to indicate that these two methods are already run in a transaction